.vib/spring-cloud-dataflow/cypress/cypress/integration/prepare-app-state.js

```diff
@@ -0,0 +1,52 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+export const importAStreamApplication = () => {
+  cy.visit('/dashboard');
+  cy.contains('button', 'Add application(s)').click();
+  cy.contains(
+    '.clr-control-label',
+    'Stream application starters for Kafka/Maven'
+  ).click();
+  cy.contains('.btn-primary', 'Import').click();
+  cy.get('.toast-container').should('contain', 'Application(s) Imported');
+  cy.get('.content-area')
+    .should('contain', 'processor')
+    .and('contain', 'sink')
+    .and('contain', 'source');
+};
+
+export const importATaskApplication = () => {
+  cy.visit('/dashboard');
+  cy.contains('button', 'Add application(s)').click();
+  cy.contains(
+    '.clr-control-label',
+    'Task application starters for Maven'
+  ).click();
+  cy.contains('.btn-primary', 'Import').click();
+  cy.get('.toast-container').should('contain', 'Application(s) Imported');
+  cy.get('[routerlink="tasks-jobs/tasks"]').click();
+  cy.contains('button', 'Create task').click();
+  cy.get('.content-area')
+    .should('contain', 'timestamp')
+    .and('contain', 'timestamp-batch');
+};
```

Comment:
As you are reusing about half of the lines with `importAStreamApplication`, do you think it would make sense to have a generic method, say `importApplication(String type)`, and act depending on the Application `type`?

Reply:
Yes, I like this suggestion a lot! Applied it.